eng/scripts/Language-Settings.ps1

```diff
@@ -227,13 +231,13 @@ function Find-dotnet-Artifacts-For-Apireview($artifactDir, $packageName)
   return $packages
 }
 
-function SetPackageVersion ($PackageName, $Version, $ServiceDirectory, $ReleaseDate, $BuildType=$null, $GroupId=$null)
+function SetPackageVersion ($ArtifactName, $Version, $ServiceDirectory, $ReleaseDate, $BuildType=$null, $GroupId=$null)
```

Comment:
Do we not have the PackageName in these calls?

Reply:
Yes, but the actually thing passed is the ArtifactName so I want to make it clear